### ISSUE TITLE
Font is too small on high res windows 10 

### ISSUE THREAD

**Issue by BettyJJ** [hiddenswitch/Spellsource-Server#421]
I am using a Surface Pro 4 coming with Windows 10. The resolution is 2736 x 1824 and resizing at 200% (both are recommended settings).  
The "Hidden Switch Launcher.exe" has no problem, but the "Spellsource" window that comes after you click "Launch" has fonts that are painfully tiny. Please see the screenshots below.
Choosing "fullscreen" does not help.
![image](https://user-images.githubusercontent.com/245282/41635884-c7c9002e-747d-11e8-8cd3-e063ba4dc19b.png)
![image](https://user-images.githubusercontent.com/245282/41638013-84baa4d4-7489-11e8-8403-cd0b8eb4e896.png)

**Comment by doctorpangloss** [comment]
Thanks for the report.

I'll patch the client to allow you to specify a scaling factor in the main menu, with a good guess as the default.

Unfortunately, Windows does not have a consistent way of querying the user's intended HiDPI scaling factory. https://stackoverflow.com/a/21450169/1757994

**Comment by BettyJJ** [comment]
@doctorpangloss After further dig on the internet, I've found a way to adjust the settings. Choosing "scaling behavior" as "System" (as in the screenshot below) will make the program display in a reasonable size. 
Yesterday I also searched around and played with the settings, but could not make it work, but today with some luck I came across the right one. =.= 
So there is no need for you to patch anymore. Thank you very much anyway. :)
![image](https://user-images.githubusercontent.com/245282/41696650-b778063c-7547-11e8-8315-039d38c858f0.png)
